example/widgetbook/pages/components/stepper_input_widgetbook.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+import 'package:widgetbook/widgetbook.dart';
+import 'package:zeta_flutter/zeta_flutter.dart';
+
+import '../../test/test_components.dart';
+import '../../utils/utils.dart';
+
+Widget stepperInputUseCase(BuildContext context) {
+  return WidgetbookTestWidget(
+    widget: ZetaStepperInput(
+      initialValue: context.knobs.intOrNull.input(label: 'Initial value'),
```

Comment:
I think it would be better if this one were just `int` rather than `intOrNull` as it could be confusing in widgetbook